launchable/commands/subset.py

```diff
@@ -293,7 +294,8 @@ def run(self):
                     rests = res.json()["rest"]
                     subset_id = res.json()["subsettingId"]
                     summary = res.json()["summary"]
-                    is_brainless = res.json()["isBrainless"]
+                    is_brainless = res.json().get("isBrainless", False)
+                    is_observation = res.json().get("observation", False)
```

Comment:
huh, we use `is` prefix for `isBrainless` to describe a boolean field and now I'm thinking using `isObservation` or `isObservationMode` is better for the consistency.

Reply:
yeah, it's old my bad... 
I prefer `isObservation` to `isObservationMode`. What do you think?

Reply:
agreed. Let's use `isObservation` here and change the payload of the request as well!

Reply:
👍

Reply:
What about `record session` [payload](https://github.com/launchableinc/cli/blob/571b00982cd5cab55f22fda87c2d2e14ee4cf1e0/launchable/commands/record/session.py#L75)? I think this should be `isObservation`.

Reply:
Ah, I agree with you! I'll fix it too.

Reply:
done https://github.com/launchableinc/cli/pull/438/commits/d66db7fb09febcabb5b13466bf6471bb2dc4dd6a